src/Interfaces/SchemaRegistryClientInterface.php

```diff
@@ -0,0 +1,15 @@
+<?php
+
+namespace Jobcloud\KafkaSchemaRegistryClient\Interfaces;
+
+interface SchemaRegistryClientInterface
```

Comment:
```suggestion
interface SchemaRegistryHttpClientInterface
```
naming suggestion, since it seems to be a http client interface